server/src/main/resources/application-local.properties

```diff
@@ -1,6 +1,6 @@
 spring.config.import=local.properties
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/kitu-dev
+spring.datasource.url=jdbc:postgresql://localhost:5432/kitu-dev?stringtype=unspecified
```

Comment:
Pitää laittaa (tai en tiedä parempaa), enum - mäppäyksiä varten. 

https://stackoverflow.com/questions/851758/java-enums-jpa-and-postgres-enums-how-do-i-make-them-work-together